build.gradle

```diff
@@ -45,8 +45,13 @@ dependencies {
     implementation 'com.fasterxml.jackson.core:jackson-databind:2.15.2'
     implementation 'com.auth0:java-jwt:4.4.0'
     implementation 'com.auth0:jwks-rsa:0.22.1'
-    implementation 'org.apache.jclouds:jclouds-allblobstore:2.5.0'
-    implementation 'org.apache.jclouds.api:filesystem:2.5.0'
+    // fat jar of https://github.com/Maxim-Gadalov/jclouds/tree/2.5.0-gcp-fix
```

Comment:
I suggested to replace your forked repo with an official repo and branch where the fat jar was built on.

Reply:
fixed